app/views/partials/_user.html.erb

```diff
@@ -0,0 +1,13 @@
+<% if user.present? %>
+<div class="user-card">
+  <div class="user-photo-box">
+    <img src="<%= user.photo %>" alt="User photo" class="user-photo">
+  </div>
+  <div class="user-details">
+    <h3 class="user-name"><%= user.name %></h3>
```

Comment:
- [ ] It would be better if the name were a link that allows the user to go to the user#show page, however, when I tried to see the show page from user 1 I got this alert: 👇🏽 

![image](https://github.com/KAYEMBAJONATHAN/Blog-app-/assets/58642949/04db4d6f-efab-4b56-94a3-e1233f22c611)

Please 🙏🏽 fix the show page to be able to see the user details.